src/debug.c

```diff
@@ -1031,6 +1031,11 @@ __attribute__((noinline)) void _serverAssert(const char *estr, const char *file,
     bugReportEnd(0, 0);
 }
 
+/* Returns the amount of client's command arguments we allow logging */
+int clientArgsToLog(const client *c) {
```

Comment:
```suggestion
int canLogClientArg(const client *c, int idx) {
    serverAssert(idx < c->arvc);
    return idx >= server.hide_client_log ? 1 : c->argc;
}
```
Ideally we would also allow printing of other arguments that aren't keys, so I think this structure of a function makes more sense. I don't think we need to necessarily implement the logic to figure out which arguments are keys though.

Reply:
Ack so `hide_client_log` would be an integer which will be set by default to be 1. To disable the configuration user will set it to -1 (or other negative value).